tdvt/tdvt/exprtests/standard/setup.string.endswith.null.txt

```diff
@@ -0,0 +1,3 @@
+ENDSWITH(str2, Null)
+ENDSWITH(str1,'')
+ENDSWITH(num0, Null)
```

Comment:
```suggestion
ENDSWITH(str1, Null)
```
I think all of these tests should use str1 (no null values) and str2 (null values). An int or num column is out of scope for the string tests.

Its possible all the same coverage can be achieved in these tests by just running test cases on str2